### PR TITLE
Fix AV in netebpfext when packet with multiple MDLs is received

### PR DESCRIPTION
**Pull request by saxena-anurag**
This fixes an issue in netebpfext where it hits a bugcheck when trying to clone an NBL whose NB has multiple MDLs.

### CODE REVIEW

File: libs/platform/kernel/ebpf_platform_kernel.c [hunk 1]
```diff
@@ -212,7 +212,7 @@ ebpf_allocate_ring_buffer_memory(size_t length)
     }
 
 #pragma warning(push)
-#pragma warning(disable : 28145) /* The opaque MDL structure should not be modified by a driver except for
+#pragma warning(disable : 28145) /* The opaque MDL structure should not be modified by a driver except for \
```
Comment (shankarseal) [inline libs/platform/kernel/ebpf_platform_kernel.c:215]: Why is this change needed?

**Review by shankarseal — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

File: libs/platform/kernel/ebpf_platform_kernel.c (hunk 1 above)
Comment (dthaler) [reply libs/platform/kernel/ebpf_platform_kernel.c:215]: I asked this on Alan's PR and he said the code formatter added it automatically.
